projects/observability/src/shared/dashboard/dashboard-wrapper/navigable-dashboard.component.ts

```diff
@@ -37,6 +40,7 @@ import { GraphQlFilterDataSourceModel } from '../data/graphql/filter/graphql-fil
   `
 })
 export class NavigableDashboardComponent implements OnChanges {
+  public readonly pageLoadingConfig: LoadAsyncConfig = { load: { loaderType: LoaderType.Logo } };
```

Comment:
Ohh... Traceable Logo as the page loader? I am not sure how it would look. 

@jake-bassett thoughts?

Reply:
It's for the cases where a page/dashboard level loader is required, specifically when hLoadAsync is placed on high level elements. For example api-enpoints and security-event pages would look like this without it:
<img width="1764" alt="rectangle-loading-page" src="https://user-images.githubusercontent.com/35982795/151559300-869546f5-e678-4fa8-bf49-9b17eb52297b.png">
(just using application-flow page as example, in reality this will have the topology loader)

With the logo on page level htLoadAsync instances, like api-enpoints and security-event, look like this:
<img width="1769" alt="security-event-logo" src="https://user-images.githubusercontent.com/35982795/151559815-ee363167-d1f4-4adf-bd77-8df7938a6774.png">

Thoughts ?

Reply:
I think, We should have some page specific loaders as well, I think that will look good.
Suppose having an application flow loader with some animations.